src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -207,6 +215,20 @@ def _migrate_dbfs_root_table(self, src_table: Table, rule: Rule, grants: list[Gr
         self._backend.execute(src_table.sql_alter_from(rule.as_uc_table_key, self._ws.get_workspace_id()))
         return self._migrate_acl(src_table, rule, grants)
 
+    def _migrate_table_in_mount(self, src_table: Table, rule: Rule, grants: list[Grant] | None = None):
+        target_table_key = rule.as_uc_table_key
+        fields = []
+        for key, value, _ in self._backend.fetch(f"DESCRIBE TABLE delta.`{src_table.location}`;"):
```

Comment:
The `src_table` could also be parquet, json, or csv right? If this PR migrate delta in mounts only, then the 
```python
if self.is_table_in_mount:
    return What.TABLE_IN_MOUNT
```
should be corrected.

Reply:
True, I'll edit this to add `self.is_delta` as a condition